addons/mail/static/src/discuss/core/public_web/discuss_sidebar_categories.xml

```diff
@@ -92,8 +92,9 @@
             <Dropdown t-if="store.discuss.isSidebarCompact" state="floating" position="'right-middle'" menuClass="'o-mail-DiscussSidebar-floatingMenu bg-view border-0 p-2 m-0 min-w-0'" manual="true">
                 <t t-call="mail.DiscussSidebarChannel.main"/>
                 <t t-set-slot="content">
-                    <div class="overflow-hidden" t-ref="floating">
-                        <span class="fw-bolder user-select-none" t-esc="thread.displayName"/>
+                    <div class="overflow-hidden text-truncate d-flex flex-column" t-ref="floating">
+                        <span class="fw-bolder user-select-none text-truncate" t-esc="thread.displayName"/>
+                        <span t-if="thread.correspondent?.persona.custom_status" class="text-muted text-truncate" t-out="thread.correspondent.persona.custom_status" />
```

Comment:
Whenever you `text-truncate`, you need `title` or equivalent to see whole text, otherwise important content might not be readable.